src/swagger-specs/segmentation.yaml

```diff
@@ -1,2359 +1,5246 @@
+      - All requests with a payload in the request body(such as POST, PUT, and PATCH calls) must include the header `Content-Type` with a value of `application/json`."
+  version: "1.0"
+servers:
+- url: //platform.adobe.io/data/core/ups
 tags:
-  - name: Estimates
-    description: Estimates provide statistical information on a segment definition, such as the projected audience size and confidence interval.
-  - name: Export jobs
-    description: Export jobs are asynchronous processes that are used to persist audience segment members to datasets.
-  - name: Previews
-    description: Previews provide paginated lists of qualifying profiles for a segment definition.
-  - name: Schedules
-    description: Create a recurring schedule to automatically run export jobs. A schedule can run at most once daily.
-  - name: Segment definitions
-    description: Segment definitions include a Profile Query Language (PQL) statement that defines which profiles will be part of an audience segment.
-  - name: Segment jobs
-    description: Segment jobs process previously established segment definitions to generate an audience segment.
-  - name: Segment search
-    description: Segment search allows users to search across multiple namespaces or get specific structural information about specified objects.
-
+- name: Audiences
+  description: Audiences are a collection of people who share similar behaviors and/or characteristics. These collections of people can be generated either by using Platform or from external sources. 
+- name: Estimates
+  description: Estimates provide statistical information on a segment definition, such as the projected audience size and confidence interval.
+- name: Export jobs
+  description: Export jobs are asynchronous processes that are used to persist audience segment members to datasets.
+- name: Previews
+  description: Previews provide paginated lists of qualifying profiles for a segment definition.
+- name: Schedules
+  description: Create a recurring schedule to automatically run export jobs. A schedule can run at most once daily.
+- name: Segment definitions
+  description: Segment definitions include a Profile Query Language (PQL) statement that defines which profiles will be part of an audience segment.
+- name: Segment jobs
+  description: Segment jobs process previously established segment definitions to generate an audience segment.
+- name: Segment search
+  description: Segment search allows users to search across multiple namespaces or get specific structural information about specified objects.
 paths:
-  '/export/jobs':
+  /audiences:
+    get:
+      tags:
+      - Audiences
+      summary: Retrieve a list of all audiences
```

Comment:
"List audiences" (you're not necessarily listing all resources in a call since they should all support filtering and pagination)